test/integration/check_navbar_links.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Checks that the links in the navbar on the homepage have the right URLs to redirect to
+ *
+ * Test cases: https://github.com/LLK/scratch-www/wiki/Most-Important-Workflows#Create_should_take_you_to_the_editor
+ */
+
+var tap=require('tap'),  
+    seleniumWebdriver = require('selenium-webdriver');
+
+/*
+ * Remove question comments after resolving them
+ */
+//how to generalize for other browsers... how will this work in saucelabs? Do I need to iterate through the drivers for each browser here?
+//chrome driver
```

Comment:
Using Sauce Connect with webdriver: https://wiki.saucelabs.com/display/DOCS/Updating+Test+Scripts+to+Use+Sauce+Connect
Here is how Travis gets set up with SauceLabs: https://docs.travis-ci.com/user/sauce-connect/

To simplify running these tests locally as well as on Travis, maybe we should just always run them in Sauce?